tradeit-android-sdk/src/main/java/it/trade/android/sdk/model/TradeItOrderDetailsParcelable.java

```diff
@@ -34,6 +34,42 @@
     TradeItOrderDetailsParcelable() {
     }
 
+    @Override
+    public String toString() {
```

Comment:
Is there a real benefit in term of performance, writing a custom `toString()` with `StringBuilder` than generating the method by IntelliJ?

Reply:
I just didn't realize it can be generated. 😱